src/django_keycloak/admin.py

```diff
@@ -60,4 +61,7 @@ def has_change_permission(self, request, obj=None):
         return False
 
 
-admin.site.register(User, UserAdmin)
+try:
+    admin.site.register(User, UserAdmin)
+except admin.sites.AlreadyRegistered:
+    logging.warning(f"Could not register Keycloak UserAdmin")
```

Comment:
Getting the logger for a file allows for more fine-grained logging levels depending on the component. This has been done in [src/django_keycloak/token.py](https://github.com/urbanplatform/django-keycloak-auth/blob/6a8af99f6c7dbff0474fe067dd03c17e047bdada/src/django_keycloak/token.py).

```py
logger = logging.getLogger(__name__)
logger.warning("...")
```